### PR TITLE
Use mb/once for relevant upload tests

### PR DESCRIPTION
**Pull request by crisptrutski**
### Description

I found the mixed selection slightly annoying, as I tend to run my REPL with only a single driver enabled.

It's also nice to run the tests exhaustively locally, as we do with our other `:mb/once` tests.

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -236,7 +236,8 @@
     djblue/portal                {:mvn/version "0.51.0"}                    ; ui for inspecting values
     hashp/hashp                  {:mvn/version "0.2.2"}                     ; debugging/spying utility
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    io.github.metabase/hawk      {:sha "59cba703e757bd94c0cc698dbf5338bf36276991"}
+    io.github.metabase/hawk      {:git/url "https://github.com/metabase/hawk.git"
+                                  :sha     "490c9b04fbf5861dbd47c0c5281dad2d113bd285"}
```
Comment (crisptrutski) [inline deps.edn:240]: The upgrade is needed for to support [per test tag exclusions](https://github.com/metabase/hawk/pull/21)

I'm not sure why the git URL is required to resolve it now, while not being so before.

**Review by calherries — APPROVED** [review]
Nice!

(looks like you need to special-case `create-csv-upload!-schema-does-not-sync-test` for h2 and mysql though. Probably using `ddl.i/format-name` will help)

File: test/metabase/upload_test.clj [hunk 2]
```diff
@@ -1224,9 +1221,8 @@
                    (= :schema-filters (keyword (:type conn-prop))))
                  (driver/connection-properties driver))))
 
-(deftest create-csv-upload!-schema-does-not-sync-test
-  ;; Just test with postgres because failure should be independent of the driver
-  (mt/test-driver :postgres
+(deftest ^:mb/once create-csv-upload!-schema-does-not-sync-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
```
Comment (calherries) [inline test/metabase/upload_test.clj:1225]: I think this test implementation is pretty dependent on postgres specifics. Maybe this is best leaving

Comment (crisptrutski) [reply test/metabase/upload_test.clj:1225]: Yeah it's failing with H2, and I wasn't sure why. I'll update the comment.

**Pull request merged**